cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1569,11 +1569,24 @@ function Install_Containerd {
   Remove-Item -Force -Recurse $tmp_dir
 }
 
+# Lookup the path of containerd config if exists, else returns a default.
+function Get_Containerd_ConfigPath {
+  $service = Get-WMIObject -Class Win32_Service -Filter  "Name='containerd'"
+  if (!($service -eq $null) -and
+      $service.PathName -match ".*\s--config\s*(\S+).*" -and
```

Comment:
Would it be better to just assume the path is hard codes or pass the config through kubeenv?

Reply:
I guess we can do so, but I feel that assuming the config location never changes isn't perfect especially with containerd. unlike docker, where its installed package always kept the binaries in well-known paths, containerd is shipped as tar and can be extracted anywhere. This makes it a bit safer, so we don't re-register service again but knowing that a certain config location is what being used. If non is found, then it'll be the default path hardcoded by containerd.